sale_partner_primeship/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds a concept of primeship for partners.
+This primeship is activated on the confirmation of a sale containing
+a primeship activation product for a customizable duration.
```

Comment:
describe what is a premiship

specify only one active primeship at the time and only one time of "type" primeship